plugins/opensslcryptoplugin/evp_p.h

```diff
@@ -8,6 +8,8 @@
 #ifndef SAILFISHCRYPTO_PLUGIN_CRYPTO_OPENSSL_EVP_P_H
 #define SAILFISHCRYPTO_PLUGIN_CRYPTO_OPENSSL_EVP_P_H
 
+#include "Crypto/cryptomanager.h"
```

Comment:
This is a C header, so probably just pass the block mode as an int to avoid having to pull in this header.
(Not that there's any need for this to be a C header, specifically, but it was copy-pasted from another C-only project.)

Reply:
Yes true. Changing this means there will be hardcoded checks like (block_mode == 3), if that's what we want.

Reply:
@chriadam Out of curiosity, where did you copy-paste it from?